### PR TITLE
Truncate list of author names

### PR DESCRIPTION
**Pull request by edmadrid**
- adds readmore.js jquery plugin to truncate list of author names on item pages, with option to view all names. 
https://github.com/jedfoster/Readmore.js

- for search results page, adds css to truncate author names with ellipses

### CODE REVIEW

File: app/assets/javascripts/readmore.js [hunk 1]
```diff
@@ -0,0 +1,342 @@
+/*!
+ * @preserve
+ *
+ * Readmore.js jQuery plugin
+ * Author: @jed_foster
+ * Project home: http://jedfoster.github.io/Readmore.js
+ * Licensed under the MIT license
```
Comment (cgalarza) [inline app/assets/javascripts/readmore.js]: Can you add a statement saying what version this is?

File: app/views/catalog/_show_default.html.erb [hunk 2]
```diff
@@ -22,6 +24,15 @@
   <% end -%>
 <% end -%>
 
+<%# Truncate author list %>
```
Comment (cgalarza) [inline app/views/catalog/_show_default.html.erb]: Can this be moved to its own js file?

File: app/assets/javascripts/readmore.js (hunk 1 above)
Comment (edmadrid) [reply app/assets/javascripts/readmore.js]: added version number to readmore.js

File: app/views/catalog/_show_default.html.erb (hunk 2 above)
Comment (edmadrid) [reply app/views/catalog/_show_default.html.erb]: done. working on dev.

**Comment by cgalarza** [comment]
@edmadrid I am concerned about using blacklight-author_ssim as a css class. :/ I want to explore if we can add out own css class names that aren't tied to the name of the solr field. I'll do a little research and get back to you.

**Comment by cgalarza** [comment]
@edmadrid I thought you could add a css class in the blacklight config, but it doesn't look like that, so for now lets just stick to this. But its something to look out for in the future.

**Pull request merged**